.rultor.yml

```diff
@@ -44,4 +44,7 @@ release:
     git push -f heroku $(git symbolic-ref --short HEAD):master
     git reset HEAD~1
     curl -f --connect-timeout 30 --retry 8 --retry-delay 60 http://beta.nerodesk.com
-    mvn clean site-deploy -Psite --errors --settings ../settings.xml --batch-mode
\ No newline at end of file
+    mvn clean site-deploy -Psite --errors --settings ../settings.xml --batch-mode
+docker:
+  as_root: true
+  image: hanswesterbeek/google-debian-oracle-jdk:8
```

Comment:
Let's use ubuntu; for example: `image: mostalive/ubuntu-14.04-oracle-jdk8` as Yegor suggested https://github.com/teamed/nerodesk/issues/46#issuecomment-77441864